tests/contrib/urllib3/test_urllib3.py

```diff
@@ -185,7 +185,7 @@ def test_connection_retries(self):
         """Tests a connection error results in error spans with proper exc info"""
         retries = 3
         try:
-            self.http.request("GET", "http://fakesubdomain." + SOCKET, retries=retries)
+            self.http.request("GET", "http://localhost:9999", retries=retries)
```

Comment:
worth being more preposterous? `:1` or `:65000` ?

Reply:
yeah i didn't really know 🤷‍♂️ anything could be potentially used unless there it a port that is reserved for such a thing (but then might not be cross platform)

Reply:
good enough imho

Reply:
9999 is technically in the _registered ports_ range 1024 to 49151. There is an example of how to retrieve an ephemeral port in `aiohttp.test_utils`

https://github.com/aio-libs/aiohttp/blob/573916f0ebea517bdb6b162bcf74078acb0dfaf9/aiohttp/test_utils.py#L83-L87

if we really want to be sure we pick an unused port, but be aware of race conditions!

Reply:
👍🏽 adding a helper for unused ports